internal/kafka/command_topic_update.go

```diff
@@ -19,10 +19,7 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/types"
 )
 
-const (
-	updateTopicConfigRestMsg    = "Updated the following configuration values for topic \"%s\"%s:\n"
-	readOnlyConfigNotUpdatedMsg = "(read-only configs were not updated)"
-)
+const updateTopicConfigRestMsg = "Updated the following configuration values for topic \"%s\"%s:\n"
```

Comment:
is this message being used as error or suggestion? it's capitalized.

Reply:
Looks like it's only used in one place though... updating!